frontend/src/scenes/notebooks/Suggestions/FloatingSuggestions.tsx

```diff
@@ -0,0 +1,45 @@
+import { Editor as TTEditor } from '@tiptap/core'
+import { FloatingMenu } from '@tiptap/react'
+import { useActions, useValues } from 'kea'
+import { insertionSuggestionsLogic } from './insertionSuggestionsLogic'
+import { isCurrentNodeEmpty } from '../Notebook/utils'
+import { useEffect } from 'react'
+import { notebookLogic } from '../Notebook/notebookLogic'
+
+export function FloatingSuggestions({ editor }: { editor: TTEditor }): JSX.Element | null {
+    const logic = insertionSuggestionsLogic()
+    const { activeSuggestion, previousNode } = useValues(logic)
+    const { setEditor } = useActions(logic)
+    const { editor: notebookEditor } = useValues(notebookLogic)
+
+    const { Component } = activeSuggestion
+
+    useEffect(() => {
+        setEditor(notebookEditor)
+    }, [notebookEditor])
+
+    return (
+        <FloatingMenu
+            editor={editor}
+            tippyOptions={{ duration: [100, 0], placement: 'left', offset: [0, 0] }}
+            className="NotebookFloatingButton"
+            shouldShow={({ editor }: { editor: TTEditor }) => {
+                if (!editor) {
+                    return false
+                }
+                if (
+                    editor.view.hasFocus() &&
+                    editor.isEditable &&
+                    editor.isActive('paragraph') &&
+                    isCurrentNodeEmpty(editor)
```

Comment:
something for next time but it seems like this can show sometimes even if it is inline rather than only at the start of the line. Not sure why...